tools/utils/Utils/ProcessRunner/PackageEditorRunner.cs

```diff
@@ -0,0 +1,454 @@
+﻿//-----------------------------------------------------------------------
+// <copyright file="PackageEditorRunner.cs" company="Microsoft">
+//     Copyright (c) Microsoft. All rights reserved.
+// </copyright>
+//-----------------------------------------------------------------------
+
+namespace Microsoft.Packaging.Utils.ProcessRunner
+{
+    using System;
+    using System.Globalization;
+    using System.IO;
+    using Microsoft.Packaging.Utils.Logger;
+
+    /// <summary>
+    ///  The base class for Package Editor tool.
+    /// </summary>
+    public sealed class PackageEditorRunner : SDKToolProcessRunner
+    {
+        /// <summary>
+        /// The prefix of test cloud stream uri.
+        /// </summary>
+        private const string CloudStreamPathPrefix = "http://cloudstreamtest?localpath=";

```

Comment:
>string CloudStreamPathPrefix = "http://cloudstreamtest?localpath="; [](start = 22, length = 67)

doesn't seem like it should be open sourced

Reply:
This is a test hook for another project. I'll leave it for now (although I don't like it).